make-rules/shared-macros.mk

```diff
@@ -906,6 +906,7 @@ PKG_MACROS +=   PERL_VERSION=$(PERL_VERSION)
 # |         12         |   2024-11-14   |
 # |         14         |   2026-11-12   |
 # |         15         |   2027-11-11   |
+# |         16         |   2028-11-09   |
 # +--------------------+----------------+
 #
 # See https://www.postgresql.org/support/versioning/
```

Comment:
Please update `PG_VERSION` below too.

Reply:
I want to wait until some users had the chance to test the new version.